pkg/server/server_credential_test.go

```diff
@@ -621,4 +621,330 @@ func TestCredentialAPI(t *testing.T) {
 		assert.False(tt, verifyResp.Verified)
 		assert.Contains(tt, verifyResp.Reason, "could not parse credential from JWT")
 	})
+
+	t.Run("Test Create Revocable Credential", func(tt *testing.T) {
```

Comment:
Here and elsewhere. It's easier to read tests when the names include the:
* The name of the method being tested.
* The scenario under which it's being tested.
* The expected behavior when the scenario is invoked.

I found it difficult to do that for this test. That's typically a symptom that it's testing multiple behaviors, so maybe it's an opportunity to split it into smaller units. Alternatively, we can leave these tests here to act more like integration tests, which are following more a user story, rather than testing a behavior. I'd personally vouch for more, smaller, and targeted unit tests. That makes maintaining them much easier.